src/models/User.ts

```diff
@@ -0,0 +1,27 @@
+import { Model, DataTypes } from "sequelize";
+import sequelize from "../data-acess/db";
+
+export class User extends Model {}
+User.init({
+    id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+    },
+    isDeleted: {
+        type: DataTypes.BOOLEAN,
+        allowNull: false,
+    },
```

Comment:
This field is redundant now. You can remove it and use the real deleting

Reply:
Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/e45fdbc193c750f092f42a6891b7d1d2c927945b commit